src/Topologies/dss_transform.jl

```diff
@@ -316,7 +316,7 @@ recv_buffer(ghost::GhostBuffer) = ghost.recv_data
 create_ghost_buffer(data, topology::Topologies.AbstractTopology) = nothing
 
 function create_ghost_buffer(
-    data::Union{DataLayouts.IJFH{S, Nij}, DataLayouts.VIJFH{S, Nij}},
+    data::Union{DataLayouts.IJFH{S, Nij}, DataLayouts.VIJFH{S, <:Any, Nij}},
```

Comment:
Why `Any`?

Reply:
I did this to avoid specializing on `Nv`, we could write it as `NV ... where {..., NV, ...}`, but `Nv` is not used anywhere, so it doesn't really contribute anything useful (aside from maybe following a more consistent pattern name?). The `Any` is because type parameters are ["values"](https://docs.julialang.org/en/v1/manual/types/#%22Value-types%22):

```julia
julia> foo(::Val{<:Int}) = 1
foo (generic function with 1 method)

julia> foo(Val(2))
ERROR: MethodError: no method matching foo(::Val{2})

Closest candidates are:
  foo(::Val{<:Int64})
   @ Main REPL[1]:1

Stacktrace:
 [1] top-level scope
   @ REPL[2]:1

julia> foo(Val(Int))
1
```